inference-engine/src/vpu/graph_transformer/src/stages/resample.cpp

```diff
@@ -10,6 +10,18 @@
 #include <set>
 #include <string>
 
+constexpr auto coordinate_transformation_mode = "coordinate_transformation_mode";
+constexpr auto mode                 = "mode";
+constexpr auto asymmetric           = "asymmetric";
+constexpr auto half_pixel           = "half_pixel";
+constexpr auto nearest_mode         = "nearest_mode";
+constexpr auto round_prefer_floor   = "round_prefer_floor";
+constexpr auto round_prefer_ceil    = "round_prefer_ceil";
+constexpr auto floor_mode           = "floor";
+constexpr auto antialias            = "antialias";
+constexpr auto factor               = "factor";
+constexpr auto typeI                = "type";
```

Comment:
I can look for another way to use these variables, declare them in some common file or something so that there is no duplication.

`typeI` is used to avoid repeating the name `type`, which stands for another variable. Although, I guess i saw it not in this file, so no reason

Reply:
done

The solution : I put all of these variables in the file which is included by Interpolate.cpp, Interp.cpp, Resample.cpp (frontend.hpp). Is it okay? There is no duplication anymore.
Name `typeInterpolate` is chosen to avoid errors like
`error: invalid use of non-static member function ‘const vpu::StageType& vpu::StageNode::type() const’`
         `auto sampleType = attrs().get<ResampleType>(type);`